### ISSUE TITLE
Bands belong to different representations

### ISSUE THREAD

**Issue by sfragkos** [quanshengwu/wannier_tools#32]
Hello Dr. Wu and WannierTools users,
Let's suppose that my material has a Dirac crossing and I want to know if these two bands belong to different irreducible representations. Is there any way to know that with WannierTools?
If this is not capable with WannierTools, do you know how can I find out with VASP?
I want to know if this Dirac crossing is symmetry protected.

**Comment by quanshengwu** [comment]
@sfragkos  
No, I am sorry that WannierTools can't give you the irreducible representations (IRs). You can use QE and Wien2k to get the IRs.
